src/main/kotlin/no/nav/familie/ba/mottak/task/VurderLivshendelseTask.kt

```diff
@@ -140,10 +140,10 @@ class VurderLivshendelseTask(
         return if (åpenOppgave != null) {
             Pair(false, åpenOppgave)
         } else {
-            val fagsak = sakClient.hentRestFagsak(personIdent)
+            val fagsak = sakClient.hentRestFagsak(fagsakPerson.fagsakId)
```

Comment:
Hele hentRestFagsak virker overflødig. Her henter man fagsak for å få fagsak.id, med å bruke fagsakId som input. Så tenker du kan slette hele hentRestFagsak